docs/spfx/viva/design/designing-card.md

```diff
@@ -55,19 +55,19 @@ All layouts come in two sizes: medium or large. Card size will determine how the
 
 Use when the heading is sufficient to communicate the key information in the card.
 
-:::image type="content" source="../../../images/viva-design/img_card_heading.jpg" alt-text="Card container and elements":::
+:::image type="content" source="../../../images/viva-design/img_card_heading.jpg" alt-text="Diagram of the Time Off task and shows the Heading relationship between the two tasks.":::
 
 #### Heading + Description​​​​​​​
 
 Use when the description can add informational value to the heading and can make it easier to understand the purpose of the card. Avoid repeating information already communicated in the heading.
 
-:::image type="content" source="../../../images/viva-design/img_card_description.jpg" alt-text="Card container and elements":::
+:::image type="content" source="../../../images/viva-design/img_card_description.jpg" alt-text="Diagram of the two Time Off tasks and showing the Heading and Description for Medium and Large tasks.":::
 
 ### Heading + Image​​​​​​​
 
 Images draw viewer's eye, so use images carefully either to create recognition of a familiar app or website, or increase overall visual prominence of the card on the dashboard. The image should meaningfully connect to the heading and/or title.
 
-:::image type="content" source="../../../images/viva-design/img_card_image.jpg" alt-text="Card container and elements":::
+:::image type="content" source="../../../images/viva-design/img_card_image.jpg" alt-text="Showing the medium and large icons showing how an image appears for the medium card layout versus how it appears for the large card layout.":::
```

Comment:
Should we start this one off with, "Diagram/Screenshot that shows the medium and large icons showing..."

Using "showing" twice in a sentence sounds awkward.